unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/Utils/WorldStateUtils.cs

```diff
@@ -129,5 +130,47 @@ static IParcelScene GetCurrentScene()
 
             return scene;
         }
+
+        public static IParcelScene CreateTestScene(LoadParcelScenesMessage.UnityParcelScene data = null)
```

Comment:
here i see another chance for improvement if static calls are removed

Reply:
This is an utility method only reserved for tests usage, don't see the real benefit of moving to a mockable class (in fact it was moved out of `SceneController` to reduce its responsibilities, something that you correctly pointed out earlier).